dspace-api/src/main/java/org/dspace/storage/rdbms/TableRowIterator.java

```diff
@@ -133,7 +134,9 @@ public void setStatement(Statement st)
      * @return - The next row, or null if no more rows
      * @exception SQLException -
      *                If a database error occurs while fetching values
+     * @deprecated You should try to pass an existing database connection to this method to prevent opening a new one.
```

Comment:
Please reference the new method (i.e. `next(Context)`) by name so that developers know what the replacement is